assets/js/googlesitekit/widgets/components/WidgetRenderer.js

```diff
@@ -39,11 +39,16 @@ const WidgetRenderer = ( { slug } ) => {
 
 	// Capitalize the "component" variable, as it is required by JSX.
 	const { component: Component, wrapWidget } = widget;
-	const widgetComponent = <Component slug={ slug } />;
 
-	return widgetComponent && wrapWidget
-		? <Widget slug={ slug }>{ widgetComponent }</Widget>
-		: widgetComponent;
+	if ( ! wrapWidget ) {
+		return <Component slug={ slug } />;
```

Comment:
Oh I see we were originally passing `slug`, but that actually shouldn't be the case. Let's remove the prop here too and just render the component without props.

Reply:
Updated.